WordPress/src/main/java/org/wordpress/android/ui/posts/PostUtils.java

```diff
@@ -132,8 +132,10 @@ public static void trackSavePostAnalytics(PostModel post, SiteModel site) {
                     AnalyticsUtils.trackWithSiteDetails(AnalyticsTracker.Stat.EDITOR_UPDATED_POST, site, properties);
                 } else {
                     properties.put("word_count", AnalyticsUtils.getWordCount(post.getContent()));
-                    properties.put("editor_source", AppPrefs.isAztecEditorEnabled() ? "aztec"
-                            : AppPrefs.isVisualEditorEnabled() ? "hybrid" : "legacy");
+                    properties.put("editor_source",
+                            AppPrefs.isGutenbergEditorEnabled() ? "gutenberg"
```

Comment:
Hmm, perhaps the flag alone might not be enough since the logic we're [currently using](https://github.com/wordpress-mobile/WordPress-Android/blob/bd246e1ddc108f52b01f3ee971fc28ffbdb4029a/WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java#L2061-L2064) doesn't guarantee that Gutenberg will be the actual editor used, right?

Reply:
sorry @hypest  didn't see the link properly, checking now

Reply:
ok making sure in d514d53d6c to reuse the same logic for both showing Gutenberg and tracking. Note that `mIsNewPost` is not available as a flag-logic per se outside of `EditPostActivity`, but the check will pass anyway as long as Gutenberg is enabled and the PostModel contains blocks. As a proxy to `mIsNewPost`, I'm using `postModel.isLocalDraft()` but I made the choice loosely knowing the other check is in place.

Reply:
This seems to work well @mzorz 🙇‍. Only thing I'd suggest is to add a comment before the `postModel.isLocalDraft()` bit as it's not too intuitive why we use it right there.

Also, `checkstyle` seems to complain on Travis.

Reply:
Style defect fixed in 102a2ceec5
added explanation in 7e09176f54
👍